hypothesis-python/tests/ghostwriter/test_expected_output.py

```diff
@@ -185,6 +186,15 @@ def divide(a: int, b: int) -> float:
                 )
             ],
         ),
+        pytest.param(
+            ("xml_etree_ElementTree", ghostwriter.magic(xml.etree.ElementTree)),
+            marks=[
+                pytest.mark.skipif(
+                    sys.version_info[:2] not in [(3, 9), (3, 10)],
+                    reason="xml.etree.ElementTree.indent new in version 3.9",
```

Comment:
Why test `xml.etree.ElementTree` at all?  We definitely need some comments to explain what error this test is defending us against.

If that's just "works for classes with classmethod(s) and staticmethod(s)" I'd prefer to use a minimal example written into this file, like the small functions-to-test above.

Reply:
It's just because it is difficult to find a classmethod in builtins I think it is more common in custom codes. I can swap it for custom code if you prefer as I really can't think of any other builtin class that has classmehtod

Reply:
swapped